<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XUMusicPlayer</class>
 <widget class="QWidget" name="XUMusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="MusicplayerCDLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="MusicplayerSlider">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>430</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayerPlaymodeButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayerSetvolumedownButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayerScanButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>490</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayernextButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayerplayButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayerpreviousButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="MusicplayerlistWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>361</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MusicplayerplayingLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MusicplayersongtimeLabel">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>450</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayerreturnButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MusicplayerSetvolumeupButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>490</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
